git/datadog_create_test.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
1- How come this script is in the `git` dir (instead of `datadog` or `monitoring` for example)?
2- where is this script executed from?

Reply:
The script is executed locally.
I can move it to the right repo

Reply:
I see... it would be good to also add a README (in the file's subdirectory) to instruct the user on how to use the script, and use the term "check" instead of "test" (as "test" can be misleading).